recipe/meta.yaml

```diff
@@ -20,18 +20,24 @@ build:
 requirements:
   host:
     - pip
-    - python >=3
+    - python
+    - wheel
+    - setuptools
   run:
-    - python >=3
+    - python >=3.7
```

Comment:
Arch-specific packages should not use `>=3.x` in their `python` run constraints. Those bounds should be automatically generated by `conda build` based on the python version fed in from cbc.yaml.